src/ig/IGExporter.ts

```diff
@@ -1125,7 +1195,9 @@ export class IGExporter {
     iniObj.template = this.config.template;
     const comment = [
       'This ig.ini was generated using the template property in sushi-config.yaml. To provide your own',
```

Comment:
This is showing up in legacy projects where `config.yaml` is used too, so we should replace `sushi-config.yaml` with `${this.configName}`